src/mono/wasm/runtime-test.js

```diff
@@ -202,8 +209,8 @@ loadScript ("mono-config.js");
 var Module = {
 	mainScriptUrlOrBlob: "dotnet.js",
 
-	print: print,
-	printErr: function(x) { print ("WASM-ERR: " + x) },
+	print,
+	printErr,
```

Comment:
What will be printed here now and are these two even needed now that they are defined above?

Reply:
This is just explicitly capturing the definitions above in Module so console.log and console.error respectively.  The pr is still a draft because I want to finish reviewing where they get used in the rest of the stack.